jfrog-cli/artifactory/commands/golang/go.go

```diff
@@ -93,3 +104,24 @@ func validatePrerequisites() error {
 	}
 	return nil
 }
+
+/*
+Resolves the dependencies from the official Go repositories and publish those dependencies to Artifactory
+*/
+func TryToDownloadAndPublish(targetRepo string, details *config.ArtifactoryDetails) error {
+	err := os.Unsetenv(goutils.GOPROXY)
+	if err != nil {
+		return errorutils.CheckError(err)
+	}
+	err = goutils.DownloadDependenciesDirectly()
+	if err != nil {
+		return err
+	}
+	// Publish the dependencies.
```

Comment:
Let's add a log message here:
*log.Info("Uploading dependencies to Artifactory...")*

Reply:
This message already printed:

[Info] Publishing package dependencies...
[Info] Publishing: gopkg.in/go-playground/validator.v8:v8.18.2 to go-local2

This message from the publish function:
	// Publish the dependencies.
	_, _, err = Publish(false, "ALL", targetRepo, "", "", "", details) --> PublishDependencies:
	log.Info("Publishing package dependencies...")